crates/algokit_utils/src/clients/app_manager.rs

```diff
@@ -377,6 +377,61 @@ impl AppManager {
         Ok(values)
     }
 
+    /// Decode box value using ABI type.
+    ///
+    /// This method takes an ABIType directly and uses it to decode the box value,
+    /// returning an ABIValue directly for simpler usage patterns that match the
+    /// TypeScript and Python implementations.
+    ///
+    /// # Arguments
+    /// * `app_id` - The application ID
+    /// * `box_name` - The box name identifier
+    /// * `abi_type` - The ABI type to use for decoding
+    ///
+    /// # Returns
+    /// An ABIValue containing the decoded box value
+    pub async fn get_box_value_from_abi_type(
```

Comment:
I was expecting that `get_box_value_from_abi_method` and `get_box_values_from_abi_method` would be removed.